### PR TITLE
Integrate container logging

### PR DESCRIPTION
**Pull request by saschagrunert**
Requires #300 

The commit e9dcc9fb5488801f953b88b3f92294388377c3ce is ready for review.

PTAL @haircommander @rphillips

### CODE REVIEW

File: conmon-rs/common/proto/conmon.capnp [hunk 1]
```diff
@@ -20,6 +20,7 @@ interface Conmon {
         bundlePath @1 :Text;
         terminal @2 :Bool;
         exitPaths @3 :List(Text);
+        logPath @4 :Text;
```
Comment (haircommander) [inline conmon-rs/common/proto/conmon.capnp:23]: Eventually, we may need to support multiple log driver/log path pairs. currently, conmon just has them specified as `log_driver:/path`, but I think we could do better. maybe we could create a LogConfig and have a vector of those LogConfigs in the request?

**Comment by haircommander** [comment]
I think this is a good start. We should make cri_logger optoinal, and have it be more generic (have a logger trait that cri logger satisfies). 

that, plus the protocol changes to allow for it to be generic

for now, let's merge and iterate (BTW: the logging tests we expect to pass pass when this is integrated into CRI-O)

**Comment by haircommander** [comment]
@rphillips LGTY?

**Review by rphillips — APPROVED** [review]

File: conmon-rs/common/proto/conmon.capnp (hunk 1 above)
Comment (saschagrunert) [reply conmon-rs/common/proto/conmon.capnp:23]: Good, following up on this in https://github.com/containers/conmon-rs/pull/315

**Pull request merged**